src/main/java/mate/controller/carcontroller/GetAllCarsController.java

```diff
@@ -0,0 +1,28 @@
+package mate.controller.carcontroller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.service.CarService;
+
+@WebServlet(urlPatterns = "/cars")
+public class GetAllCarsController extends HttpServlet {
+    private static final Injector injector =
+            Injector.getInstance("mate");
```

Comment:
```suggestion
    private static final Injector injector = Injector.getInstance("mate");
```

Reply:
if the line is not longer than 100 symbols, don't break it. fix this wherever possible.